### PR TITLE
Move config files around to clean up root

### PR DESCRIPTION
**Pull request by AndrewGable**
Please review @tgolen 
cc @marcaaron 

### Details
Moving the various config files around to make the root have less files.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/146267

### Tests
Make sure all platforms build successfully

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

### CODE REVIEW

File: src/CONFIG.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const expensifyURLRoot = Config.USE_NGROK === 'true' && Config.NGROK_URL
     : expensifyCom;
 
 export default {
+    APP_NAME: 'ExpensifyCash',
+    DISPLAY_NAME: 'Expensify Cash',
```
Comment (tgolen) [inline src/CONFIG.js:28]: It doesn't look like this is ever used?

**Review by tgolen — CHANGES_REQUESTED** [review]
Did you want to move the eslint file back into the root to make it easier on the lives of our contributors?

**Comment by AndrewGable** [comment]
@marcaaron has a PR [here](https://github.com/Expensify/Expensify.cash/pull/1103) that is handling the eslint file.

File: src/CONFIG.js (hunk 1 above)
Comment (AndrewGable) [reply src/CONFIG.js:28]: Yeah, good point, I will remove!

**Review by tgolen — APPROVED** [review]

**Pull request merged**